visual_servoing:  
  ros_control:
    rate: 14
  image:
    show_img: true
    roi_initial: 320
    threshold: [0.025,0.2]
    min_area: 5
    roi_range: 40
    plant_radius: 1.0
  control:
    # gais: [kRho, KTheta, kV, kIRho, kITheta,kDRho,kDTheta]
    # gains: [-3.2,0.01,0.1,0.007,0.0010,0.8,0.05]
    # Solarbot
    #gains: [-2.8,0.006,1.3,0.007,0.0010,0.8,0.05]
    # Soybot (simulator)
    gains: [-0.5,0.006,0.5,0.007,0.0010,0.8,0.05]
    lin_vel: 0.5
    control_pi: false
